javascript/javascript2/week1/spirit-animal-name/spirit-animal-name-generator.js

```diff
@@ -0,0 +1,40 @@
+console.log('Hello! Want to know your animal spirit name!');
+const array = [
```

Comment:
Array is not a good name :). Try
```suggestion
const funnyAnimalNames= [
```